api/src/main/java/com/nicknathanjustin/streamercontracts/exampleapi/ExampleApiController.java

```diff
@@ -28,16 +30,16 @@ public String getRestrictedAPI() {
         return "wow! you hit a restricted endpoint. good job!";
     }
 
-    @RequestMapping(method = RequestMethod.GET)
-    public String getExample() {
-        return exampleService.getExampleValue(UUID.randomUUID());
-    }
-
     @RequestMapping(path = "{id}", method = RequestMethod.GET)
     public String getExampleById(@PathVariable("id") @NonNull final UUID uuid) {
         return exampleService.getExampleValue(uuid);
     }
 
+    @RequestMapping(method = RequestMethod.GET)
+    public void redirect(HttpServletResponse response) throws IOException {
+        response.sendRedirect("http://localhost:3000/profile");
```

Comment:
this needs to be configurable so we can redirect differently based on environment (beta wouldnt redirect to localhost). We handle different configuration for environments by changing the "application-{env}.yml" files. So application.yml needs these lines

login:
  redirect: http://localhost:3000/profile

Then application-beta.yml needs these lines:

login:
  redirect: streamercontracts.co.uk

Then replace the hardocded url with those values by reading the config value with spring

...Fix this if you want, but dont spend too much time worrying about it. I can handle making that change later.